contrib/99designs/gqlgen/option.go

```diff
@@ -51,6 +53,21 @@ func WithServiceName(name string) Option {
 	}
 }
 
+// SkipFieldsForIntrospectionQuery skips creating spans for fields when the operation name is IntrospectionQuery.
+func SkipFieldsForIntrospectionQuery(skip bool) Option {
+	return func(cfg *config) {
+		cfg.skipFieldsForIntrospectionQuery = skip
+	}
+}
+
+// SkipFieldsWithTrivialResolver skips creating spans for fields that have a trivial resolver.
+// For example, a field resolved from an object w/o requiring a custom method is considered trivial.
+func SkipFieldsWithTrivialResolver(skip bool) Option {
+	return func(cfg *config) {
+		cfg.skipFieldsWithTrivialResolver = skip
+	}
+}
+
```

Comment:
```suggestion
// WithoutTraceInstrospectionQuery skips creating spans for fields when the operation name is IntrospectionQuery.
func WithoutTraceInstrospectionQuery() Option {
	return func(cfg *config) {
		cfg.traceIntrospectionQuery = false
	}
}

// WithoutTraceTrivialResolvedFields skips creating spans for fields that have a trivial resolver.
// For example, a field resolved from an object w/o requiring a custom method is considered trivial.
func WithoutTraceTrivialResolvedFields() Option {
	return func(cfg *config) {
		cfg.traceTrivialResolvedFields = false
	}
}

```

I think its important to keep the existing naming convention, as it has some benefits like making easier to find all the available options from the IDE.

Also, the `skip bool` parameter shouldn't be necessary as its implicit from the `Without`.